extensions/2.0/Khronos/KHR_texture_ktx/README.md

```diff
@@ -0,0 +1,262 @@
+This extension does not define how to output the higher dynamic range result buffer to display or to a lower precision target buffer.  
+
+## glTF Schema Updates
+
+The `KHR_texture_ktx` extension is added to the `textures` object and specifies a `source` property that points to the index of the `image` which defines a reference to the KTX v2 image.
+
+The following glTF will load `image.ktx2`
+
+```json
+{
+    "asset": {
+        "version": "2.0"
+    },
+    "extensionsUsed": [
+        "KHR_texture_ktx"
+    ],
+    "textures": [
+        {
+            "source": 0,
+            "extensions": {
+                "KHR_texture_ktx": {
+                    "source": 1
+                }
+            }
+        }
+    ],
+    "images": [
+        {
+            "uri": "image.png"
+        },
+        {
+            "uri": "image.ktx2"
+        }
+    ]
+}
+```
+
+When used in the glTF Binary (GLB) format the `image` that points to the KTX v2 resource uses the `mimeType` value of `image/ktx2`.
+
+```json
+{
+    "asset": {
+        "version": "2.0"
+    },
+    "extensionsUsed": [
+        "KHR_texture_ktx"
+    ],
+    "textures": [
+        {
+            "source": 0,
+            "extensions": {
+                "KHR_texture_ktx": {
+                    "source": 1
+                }
+            }
+        }
+    ],
+    "images": [
+        {
+            "mimeType": "image/png",
+            "bufferView": 1
+        },
+        {
+            "mimeType": "image/ktx2",
+            "bufferView": 2
+        }
+    ]
+}
+```
+
+### Using Without a Fallback
+
+To use KTX v2 image without a fallback, define `KHR_texture_ktx` in both `extensionsUsed` and `extensionsRequired`. The `texture` object will then have its `source` property omitted as shown below.
+
+```json
+{
+    "asset": {
+        "version": "2.0"
+    },
+    "extensionsUsed": [
+        "KHR_texture_ktx"
+    ],
+    "extensionsRequired": [
+        "KHR_texture_ktx"
+    ],
+    "textures": [
+        {
+            "extensions": {
+                "KHR_texture_ktx": {
+                    "source": 0
+                }
+            }
+        }
+    ],
+    "images": [
+        {
+            "uri": "image.ktx2"
+        }
+    ]
+}
+```
+
+### JSON Schema
+
+[texture.KHR_texture_ktx.schema.json](schema/texture.KHR_texture_ktx.schema.json)
+
+## Color Texture Sources
+
+When this extension is used it is allowed to provide color textures that does not use sRGB transfer function for texture sources refered to by material source objects.
+In the core spec this affects baseColorTextureand emissiveTexture, in extensions it will affect any color texture source.  
+This means that when this extension is used a texture that stores color values shall not be sRGB encoded, if it contains higher range than 8 bits per color component.  
```

Comment:
Let's rephrase it in a more normative way (also shorter):
> Texture data provided by this extension is always encoded using linear transfer function. When this extension is used with material texture slots that expect sRGB-encoded values, only the fallback 8-bit texture (when present) uses sRGB encoding.
> > **Implementation Note**: For the core glTF 2.0 specification, these texture slots include `emissiveTexture` and `baseColorTexture`; for material extensions, it's usually texture slots named `*ColorTexture`.

Reply:
Great, thanks @lexaknyazev Updated - resolving this conversation.